migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs

```diff
@@ -59,6 +59,17 @@ impl SqlRenderer for PostgresFlavour {
                 })
                 .collect();
 
+            if stmts.len() > 1 {
+                let warning = indoc::indoc! {
+                    r#"-- This migration adds more than one variant into an enum.
+                       -- With PostgreSQL versions 11 and earlier, this is not possible
+                       -- in one migration. Either split them into separate files, or run
```

Comment:
It's not clear what splitting it into separate files means from a practical POV. I guess it means you add one value, create a one migration, add another value, create another migration, etc. 

Correct?